addons/mail/static/src/models/attachment_list/attachment_list.js

```diff
@@ -5,7 +5,7 @@ import { many2many, many2one, one2many, one2one } from '@mail/model/model_field'
 import { clear, insertAndReplace, replace } from '@mail/model/model_field_command';
 
 registerModel({
-    name: 'mail.attachment_list',
+    name: 'AttachmentList',
```

Comment:
While we are at it this should be renamed to `AttachmentListView` actually.

Reply:
But I guess that might be for another PR, as renaming the concept will lead to changing field name etc. Which is out of scope of what we are trying to do here.

Reply:
Noted, I'll do that once everything is merged.